csharp/CWE-502/UnsafeDeserialization.ql

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
What is the difference between [this query](https://github.com/advanced-security/codeql-queries/blob/main/csharp/CWE-502/UnsafeDeserializationAudit.ql) and this?

Reply:
This query is needed to pull in the custom library.  The library is where I am working through pulling in additional flow/taint steps for this particular scenario.  I created this as a draft PR as it's not ready for prime time yet.